lib/commands/connect/export.js

```diff
@@ -15,17 +15,21 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let url = '/api/v3/connections/' + connection.id + '/actions/export';
-    let response = yield api.request(context.auth.password, 'GET', url)
+    let connection, response;
```

Comment:
I had wondered about writing to stdout when I first started this, but I opted to just keep existing behavior for now. We have a few options:

1. Always write to a file, ignoring stdout (what we have today)
2. Always write to stdout, expecting users to pipe to a file if they need it
3. Accept `-` as a filename to indicate writing to stdout

I'm tempted to lean toward option 2. It feels the most Unix-y, but there's plenty of precedent for `-` as an alias for stdin/stdout. Thoughts?